src/integrationTest/java/com/mongodb/hibernate/query/select/SortingSelectQueryIntegrationTests.java

```diff
@@ -60,7 +61,7 @@ private static List<Book> getBooksByIds(int... ids) {
                 .mapToObj(id -> testingBooks.stream()
                         .filter(c -> c.id == id)
                         .findFirst()
-                        .orElseThrow(() -> new IllegalArgumentException("id does not exist: " + id)))
+                        .orElseThrow(() -> fail("id does not exist: " + id)))
```

Comment:
Thank you for finding and changing this one also. Could we please also replace `findFirst` with `findAny` here, in `SimpleSelectQueryIntegrationTests`, `LimitOffsetFetchClauseIntegrationTests`, similarly to https://github.com/mongodb/mongo-hibernate/pull/125#discussion_r2442725604?